### ISSUE TITLE
Using image builder for our builder images

### ISSUE THREAD

**Issue by FrostyX** [fedora-copr/copr#2701]
I swear that title makes sense.

Infra discussion: 

- https://lists.fedoraproject.org/archives/list/infrastructure@lists.fedoraproject.org/thread/SP4JNDFAS7BVCJEXMGE4KQBURNJGSGJB/

Arch specific images into quay.io:

- https://gitlab.com/ondrejbudai/hello-kubernetes/-/blob/main/.gitlab-ci.yml
- https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
- https://bugzilla.redhat.com/show_bug.cgi?id=2040685

**Comment by praiskup** [comment]
Yeah!  Any simplification in the semi-automated [Copr workflow](https://docs.pagure.org/copr.copr/how_to_upgrade_builders.html) would be awesome.

**Comment by xsuchy** [comment]
Additional arches in imagebuilder is tracked in https://issues.redhat.com/browse/COMPOSER-1706
